notebooks/278-kosmos2-multimodal-large-language-model/278-kosmos2-multimodal-large-language-model.ipynb

```diff
@@ -0,0 +1,897 @@
+{
```

Comment:
<p>I think some schema or explanation needed here for understanding from which parts model consists of</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1483/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Some explanation was added